### PR TITLE
add: Additional field to Endpointview entity

### PR DESCRIPTION
**Pull request by SebastianOpriel**
In order to use the additional field in Endpoints, the Endpointview is adjusted.

### CODE REVIEW

**Review by ghost — CHANGES_REQUESTED** [review]
can we add a test for the new field, e.g. in GenericEndpointViewAssemblerTest (example see ArtifactViewAssemblerTest)

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [Unreleased]
+
+### Added
+- Add additional field to `EndpointView`.
```
Comment (ghost) [inline CHANGELOG.md:8]: GenericEndpointView

**Comment by tmberthold** [comment]
_needs to be reopend directly in this repo since sovity repo not being a fork of this fork_

**Comment by ghost** [comment]
_needs to be reopend directly in this repo since sovity repo not being a fork of this fork_

**Comment by ghost** [comment]
will put in on my TODO list

**Pull request closed without merging**